qa/rpc-tests/p2p-compactblocks.py

```diff
@@ -145,8 +145,12 @@ def build_block_on_tip(self, node, segwit=False):
         mtp = node.getblockheader(tip)['mediantime']
         block = create_block(int(tip, 16), create_coinbase(height + 1), mtp + 1)
         block.nVersion = 4
+        if height > 1:
+            blocktemplate = node.getblocktemplate({'rules':['segwit','auxpow']})
+        else:
+            blocktemplate = []
```

Comment:
I think you mean `{}` here. Setting to `[]` makes it an array.

Reply:
Thanks. Corrected.